README.md

```diff
@@ -1,55 +1,25 @@
 # Nova Tool for Staff to Make Bookings
 
-
-[![Latest Version on Packagist](https://img.shields.io/packagist/v/tipoff/staff-bookings.svg?style=flat-square)](https://packagist.org/packages/tipoff/staff-bookings)
-![Tests](https://github.com/tipoff/staff-bookings/workflows/Tests/badge.svg)
-[![Total Downloads](https://img.shields.io/packagist/dt/tipoff/staff-bookings.svg?style=flat-square)](https://packagist.org/packages/tipoff/staff-bookings)
```

Comment:
Let's leave the badges on every package.

Reply:
a3fef73999c2db01fcbcecb29d538ede3cf43109